releases/release-1.19/README.md

```diff
@@ -0,0 +1,94 @@
+# Kubernetes 1.19
+
+
+#### Links
+
+* [This document](https://git.k8s.io/sig-release/releases/release-1.19/README.md)
+* [Release Team](https://git.k8s.io/sig-release/releases/release-1.19/release_team.md)
+* [Meeting Minutes](http://bit.ly/k8s119-releasemtg) (join [kubernetes-sig-release@] to receive meeting invites)
+* [v1.19 Release Calendar][k8s119-calendar]
+* Contact: [#sig-release] on slack, [kubernetes-release-team@] on e-mail
+* [Internal Contact Info]() (accessible only to members of [kubernetes-release-team@])
+
+#### Tracking docs
+
+* [Enhancements Tracking Sheet](http://bit.ly/k8s-1-19-enhancements)
+* Bug Triage Tracking Sheet: TODO
+* CI Signal Report: TODO
+* [Retrospective Document][Retrospective Document]
+* [kubernetes/sig-release v1.19 milestone](https://github.com/kubernetes/kubernetes/milestone/46)
+
+#### Guides
+
+* [Targeting Issues and PRs to This Milestone](https://git.k8s.io/community/contributors/devel/sig-release/release.md)
+* [Triaging and Escalating Test Failures](https://git.k8s.io/community/contributors/devel/sig-testing/testing.md#troubleshooting-a-failure)
+
+## TL;DR
+
+The 1.19 release cycle is proposed as follows:
+
+- **Monday, April 13**: Week 1 - Release cycle begins
+- **Tuesday, May 5**: Week 4 - [Enhancements Freeze]
+- **Thursday, June 11**: Week 9 - [Code Freeze]
+- **Tuesday, June 23**: Week 11 - Docs must be completed and reviewed
+- **Tuesday, June 30**: Week 12 - Kubernetes v1.19.0 released
+
+
+## Timeline
+
+
+| **What** | **Who** | **Apr** | **May** | **Jun** | **Jul** | **WEEK** | **CI SIGNAL** |
```

Comment:
It has been mentioned in the past that
```
| **What** | **Who** | **When* | **WEEK** | **CI SIGNAL** |
```
is better than
```
| **What** | **Who** | **Apr** | **May** | **Jun** | **Jul** | **WEEK** | **CI SIGNAL** |
```

specially for mobile

Reply:
Ohhhh - I like this much better, too (and it'll help the next release lead)